src/crimson/os/seastore/async_cleaner.h

```diff
@@ -1574,7 +1595,7 @@ class SegmentCleaner : public SegmentProvider, public AsyncCleaner {
   BackgroundListener *background_callback = nullptr;
 
   // TODO: drop once paddr->journal_seq_t is introduced
-  SegmentSeqAllocatorRef ool_segment_seq_allocator;
+  SegmentSeqAllocator *ool_segment_seq_allocator;
```

Comment:
It looks like EPM owns the actual object and we use it here with the assumption that it's non-null.  Since it's mandatory and we assume it's non-null, this should be a reference rather than a pointer.